zulipterminal/ui_tools/views.py

```diff
@@ -564,13 +564,18 @@ def users_view(self, users: Any=None) -> Any:
                 continue
             unread_count = (self.view.model.unread_counts['unread_pms'].
                             get(user['user_id'], 0))
+            if self.view.model.user_id == user['user_id']:
+                postfix_txt_markup = ('light grey', u" (you)")
```

Comment:
Generally we tend to use style names so that this works better across themes. Was there a reason you picked this color? I saw you did have a theme there at some point, so perhaps just leave that addition, or use an existing name if you can find one that isn't too misleading?

Note that previously your style would not have worked on a transparent background terminal, as it specified the background color to be black.